task.sql

```diff
@@ -1 +1,37 @@
-# Write your SQL code for the database creation here. Good luck! 
+CREATE DATABASE ShopDB;
+USE ShopDB;
+
+CREATE TABLE Products (
+    ID INT AUTO_INCREMENT,
+    Name VARCHAR(50),
+    Description VARCHAR(100),
+    Price DECIMAL(10, 2),
+    WarehouseAmount INT,
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Customers (
+		ID INT AUTO_INCREMENT,
+		FirstName VARCHAR(50),
+		LastName VARCHAR(50),
+		Email VARCHAR(100),
+		Address VARCHAR(100),
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Orders (
+		ID INT AUTO_INCREMENT,
+		CustomerID INT NULL,
```

Comment:
Consider making the `CustomerID` column in the `Orders` table NOT NULL if every order must be associated with a customer. This will enforce data integrity by ensuring that each order is linked to a valid customer.